packages/react-components/react-breadcrumb/src/components/Breadcrumb/useBreadcrumbStyles.ts

```diff
@@ -7,26 +7,36 @@ export const breadcrumbClassNames: SlotClassNames<BreadcrumbSlots> = {
   list: 'fui-Breadcrumb__list',
 };
 
+const useListClassName = makeResetStyles({
+  listStyleType: 'none',
+  display: 'flex',
+  alignItems: 'center',
+  margin: 0,
+  padding: 0,
+});
+
 /**
  * Styles for the root slot
  */
 const useStyles = makeStyles({
   root: {},
-  list: {
-    listStyleType: 'none',
-    display: 'flex',
-    alignItems: 'center',
-  },
+  list: {},
```

Comment:
NIT:
Is this open style object be changed in a PR later? If not, even though it has no effect, it might be better to remove. :)

Reply:
I removed it. If we need it, I'll add it